config/sync/core.entity_view_mode.user.teaser.yml

```diff
@@ -0,0 +1,10 @@
+uuid: e131451b-55e1-41ac-8253-42673b1ed409
+langcode: en
+status: true
+dependencies:
+  module:
+    - user
+id: user.teaser
+label: Teaser
+targetEntityType: user
+cache: true
```

Comment:
I think you should rename this view mode to comment teaser, because teaser on it's own is too vague

Reply:
Done.